packages/graphql_jsoo_client/graphql_jsoo_client.0.1.1/opam

```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "GrpahQL over WebSocket client"
+description:
+  "A GraphQl client that can be compiled using jsoo to access GraphQl servers using websocket. It was created with the main purpose of using it in combination with the Dream web-framework."
+maintainer: ["Hans Ole Rafaelsen"]
+authors: ["Hans Ole Rafaelsen"]
+license: "MIT"
+tags: ["graphql" "jsoo" "websockets"]
+homepage: "https://github.com/hansole/graphql_jsoo_client"
+bug-reports: "https://github.com/hansole/graphql_jsoo_client/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.13"}
+  "dream"
+  "graphql_ppx"
+  "cohttp-lwt-jsoo"
+  "js_of_ocaml-ppx"
+  "ppx_yojson_conv"
+  "uuidm"
+  "ppx_typed_fields"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/hansole/graphql_jsoo_client.git"
```

Comment:
```suggestion
x-ci-accept-failures: [
  "oraclelinux-9"
]
dev-repo: "git+https://github.com/hansole/graphql_jsoo_client.git"
```

Reply:
This will take care of the failure

Reply:
Thanks. Is the correct workflow that I delete this failed request, makes a new fixed version of the package with a new tag, and create a new request for that version?

Reply:
No, no need to do anything. You can fix the metadata on your main branch without retagging ir rereleasing, opam uses the opam file published here for the compilation of the release